rfc/drone-ci.md

```diff
@@ -0,0 +1,227 @@
+# Drone CI Spinnaker Stage Type
+| | |
+|-|-|
+| **Status**     | **Proposed**, Accepted, Implemented, Obsolete |
+| **RFC #**      | ***TBD*** |
+| **Author(s)**  | Victor Odusanya (`@Odusanya18`), Cameron Motevasselani (`@link108`) |
+| **SIG / WG**   | CI SIG |
+
+## Overview
+
+This proposal seeks to enable Drone CI Support for Spinnaker. This proposal takes advantage of plugability of CI APIs in `igor` and 	`orca` and implementation consists additions to `deck`, 	`orca` and `igor`. This is to allow for seamless Drone CI integration with existing Spinnaker setups.
```

Comment:
A new CI will also require changes to Echo to define the trigger and how it will process what events from echo to actually trigger pipelines off of.